frontend/src/router/guards.js

```diff
@@ -135,7 +135,7 @@ function ensureDataLoaded (store, localStorage) {
             deactivatedReconciliation: isAdmin,
             hideTicketsWithLabel: isAdmin
           }
-          const shootListFilters = defaults(localStorage.getObject('shootListFilter'), defaultFilter)
+          const shootListFilters = defaults(localStorage.getObject('shootList_filter'), defaultFilter)
```

Comment:
What is the reason behind this kind of renaming?

Reply:
to be consistent with `shootList_options`

Reply:
project/core/shoot/cli-gcp/terminal
projects/shoot-editor/warning

project/core/shoot-list/options
projects/shoot-list/options

project/core/shoot-list/selected-columns
projects/shoot-list/selected-columns

project/_all/shoot-list/filter

Reply:
I have adapted all keys, except for the terminals. Let's talk about that